Gulpfile.js

```diff
@@ -39,6 +42,22 @@ gulp.task('jslint', function() {
     .pipe(eslint.format());
 });
 
+// Min js files
+gulp.task('uglify', ['js'], function() {
+  gulp.src('./public/js/app.js')
+  .pipe(uglify())
+  .pipe(concat('app.min.js'))
+  .pipe(gulp.dest('./public/js'))
+});
+
+// Min css files
+gulp.task('minify-css', ['sass'], function() {
+  gulp.src('./public/css/main.css')
+  .pipe(cleanCSS())
+  .pipe(concat('main.min.css'))
+  .pipe(gulp.dest('./public/css'))
+})
+
```

Comment:
I'd create a "production" task that does 'uglify' and 'minify-css' so we can have a new script in the package.json to build the production files. What do you think? @Oliphaunte